tests/test_cms_config.py

```diff
@@ -90,6 +90,28 @@ def test_handle_content_admin_classes(self):
             admin.site._registry[PollContent].__class__.mro()
         )
 
+    def test_is_content_model_versioned(self):
+        """Test that is_content_model_versioned returns True for
+        content model that's versioned
+        """
+        extensions = VersioningCMSExtension()
+        cms_config = Mock(
+            spec=[], djangocms_versioning_enabled=True,
+            versioning=[VersionableItem(content_model=PollContent, grouper_field_name='poll')])
+        extensions.handle_versioning_setting(cms_config)
```

Comment:
As below, I suggest you not use `handle_versioning_setting` for data set up. You can set it up like this instead:
```
extensions = VersioningCMSExtension()
extensions.versionables = [VersionableItem(content_model=PollContent, grouper_field_name='poll')]
```